app/src/main/res/values/strings.xml

```diff
@@ -377,9 +377,9 @@
     <string name="keyboard_key_vibrate_title">Vibratory keypresses</string>
     <string name="keyboard_key_sound_title">Audible keypresses</string>
     <string name="autofill_auto_search_title">Auto search</string>
-    <string name="autofill_auto_search_summary">Automatically suggest search results from the web domain or application Id</string>
+    <string name="autofill_auto_search_summary">Automatically suggest search results from the web domain or app ID</string>
     <string name="allow_no_password_title">Allow no master key</string>
-    <string name="allow_no_password_summary">Enable the \"Open\" button if no credentials are selected</string>
+    <string name="allow_no_password_summary">Shows \"Open\" button if no credentials are selected</string>
```

Comment:
Same problem as before: Enable is not Shown. A disable button is grey.

Reply:
Aaaah, I get it.
I think this should be "Activates the" then?

Reply:
"Allows pressing" is OK for me